extension/src/NABfunctions.ts

```diff
@@ -82,18 +82,19 @@ export async function copySourceToTarget() {
 
 export async function findNextUnTranslatedText() {
     console.log('Running: FindNextUnTranslatedText');
-    //let workspaceSettings = Settings.GetAllSettings(null);
     let foundAnything: boolean = false;
     try {
         if (vscode.window.activeTextEditor) {
             if (vscode.window.activeTextEditor.document.uri.fsPath.endsWith('.xlf')) {
                 foundAnything = await LanguageFunctions.findNextUnTranslatedText(true);
+                if (!foundAnything) {
+                    removeCustomNotes(vscode.window.activeTextEditor.document.uri);
+                }
             }
         }
         if (!foundAnything) {
             foundAnything = await LanguageFunctions.findNextUnTranslatedText(false);
         }
```

Comment:
I think this needs to be moved further down...
```suggestion
            }
        }
        if (!foundAnything) {
            foundAnything = await LanguageFunctions.findNextUnTranslatedText(false);
            if (!foundAnything) {
                removeCustomNotes(vscode.window.activeTextEditor.document.uri);
            }
        }
```

Reply:
Seeing it now, I think my implementation of the suggestion is flawed. My idea was that the `removeCustomNotesFromFile` should run when searching the current document. This wouldn't work if nothing is found in any xlf file meaning no xlf-file would be opened to become the activeTextEditor. If I'm not mistaken the suggestion would still be open to this missed call right? 

I think the call to `removeCustomNotesFromFile` needs to be moved to `LanguageFunctions.findNextUnTranslatedText` since we never know which file is searched if in this function if it's not the `activeTextEditor`. Back to the drawing board.